Add new logserver rpc port to Vespa model

bjorncs opened the pull request:
FYI @toregge @geirst

bjorncs committed:
Add new logserver rpc port to Vespa model

arnej27959 reviewed (approved):
overall looks good, two very minor comments

geirst commented on logd/src/main/resources/configdefinitions/logd.def:
```diff
@@ -7,10 +7,13 @@ stateport int default=0
-## Port to contact the logserver on.
-logserver.port int default=5821
+## RPC port of logserver.
+logserver.rpcPort int default=5821
```
I would name this rpcport to align with naming elsewhere in the file (and def files used only in cpp code)

bjorncs committed:
Misc fixes

Keep previous default port for legacy protocol. Rename parameter
'rpcPort' => 'rpcport'. Remove 'logtp' from port suffices.

geirst reviewed (approved):
👍
